Documentation/Blazorise.Docs/Pages/Docs/Extensions/Autocomplete/AutocompletePage.razor

```diff
@@ -325,4 +325,16 @@
     <DocsAttributesItem Name="Virtualize" Type="bool" Default="false">
         Gets or sets whether the Autocomplete will use the <Code>Virtualize</Code> functionality.
     </DocsAttributesItem>
+    <DocsAttributesItem Name="TextChanged" Type="EventCallback<string>">
+        Occurs after text has changed.
+    </DocsAttributesItem>
+    <DocsAttributesItem Name="KeyDown" Type="EventCallback<KeyboardEventArgs>">
```

Comment:
We have all these new events KeyDown, OnFocus, and Blur, but none is directly mentioning which part of the Autocomplete it behaves to. Maybe we need to name them all:

- TextKeyDown, TextFocus, and TextBlur, or
- SearchKeyDown, SearchFocus, and SearchBlur, or
- InputKeyDown, InputFocus, and InputBlur

We will most possibly add more APIs in the future that will control parts of the Autocomplete so we need to decide on naming now.

Reply:
Like any other component we have, the events pertain to the shown input... the shown input for an autocomplete is a text box... any of these interactions I have with that visible input will trigger the events as expected.
I can't see how it isn't clear... but if you feel like it isn't, we can rename if you'd like. 

I like the `Search` prefix. Since we already have a `Search` / `SearchChanged` parameters.

Reply:
Don't forget that in multiple mode, the input will resize accordingly to available space. 

![image](https://github.com/Megabit/Blazorise/assets/900302/161d36f4-62bb-4dce-bd75-6c142f7a32a8)

Proceed with Search as it makes the most sense.